weatherflow2mqtt/weatherflow_mqtt.py

```diff
@@ -550,21 +549,11 @@ def _setup_sensors(self, device: WeatherFlowDevice) -> None:
             sensor_id = sensor.id
             sensor_event = sensor.event
 
-            # Don't add the Weather Sensor if forecast disabled
-            if self.forecast is None and sensor_event == EVENT_FORECAST:
-                _LOGGER.debug(
-                    "Skipping Forecast sensor %s",
-                    sensor_event,
-                )
-                continue
-
```

Comment:
The weather sensor was moved out of DEVICE_SENSORS, so this code would never fire